bolts-tasks/src/main/java/bolts/ExecutorException.java

```diff
@@ -0,0 +1,13 @@
+package bolts;
+
+/**
+ * This is a wrapper class for emphasizing that task failed due to bad Executor, rather than the continuation block it self.
+ * <p/>
+ * Created by yunarta on 22/12/15.
+ */
+public class ExecutorException extends RuntimeException {
+
+    public ExecutorException(Exception e) {
+        super("Continuation failed to executed due to exception raised in the Executor", e);
```

Comment:
s/executed/execute/g

or maybe even simplify this to "An exception was thrown by a Executor"